cmd/fleetctl/apply.go

```diff
@@ -0,0 +1,164 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"github.com/kolide/fleet/server/kolide"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+type specMetadata struct {
+	Kind    string      `json:"kind"`
+	Version string      `json:"apiVersion"`
+	Spec    interface{} `json:"spec"`
```

Comment:
I'd prefer to make this `json.RawMessage`, or leave it out so that it doesn't have to be parsed multiple times.